### PR TITLE
Improve composer consistency

### PR DESCRIPTION
**Pull request by ghost**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

This PR addresses issue #2172.
Todo list:
- [x] unclutter sylius/sylius composer.json
- [x] check for version discrepancies
- [x] try to avoid using "minimal-stability": "dev"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,48 +20,63 @@
...
         "sensio/distribution-bundle":           "2.3.*",
         "stof/doctrine-extensions-bundle":      "1.1.*",
-        "symfony-cmf/block-bundle":             "~1.0",
-        "symfony-cmf/content-bundle":           "~1.0",
-        "symfony-cmf/menu-bundle":              "~1.0",
-        "symfony-cmf/routing-bundle":           "~1.0",
-        "symfony-cmf/create-bundle":            "~1.0",
-        "symfony-cmf/media-bundle" :            "~1.0",
+        "swiftmailer/swiftmailer":              "~5.0",
+        "sylius/cart-bundle":                   "0.13.*@dev",
```
Comment (adamelso) [inline composer.json]: Why was this added here? :) Unfortunately it's preventing me from installing forks for testing new features.

Comment (mtthb) [inline composer.json]: Oh you're right, this certainly should not be there.
